common/txmgr/broadcaster.go

```diff
@@ -686,7 +686,11 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) hand
 		//
 		// In all cases, the best thing we can do is go into a retry loop and keep
 		// trying to send the transaction over again.
-		return errors.Wrapf(err, "retryable error while sending transaction %s (tx ID %d)", attempt.Hash.String(), etx.ID), true
+		if err != nil {
```

Comment:
Looks like this is supposed to return an error if `nextSequence.Int64() <= (*etx.Sequence).Int64()`. I don't think we need to add this error check since err might not be nil but we still want to return the "retryable error" still.